src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/CollectionRules/Triggers/ManualTrigger.cs

```diff
@@ -60,11 +62,18 @@ private void NotifyHandler(object sender, EventArgs args)
 
     internal sealed class ManualTriggerService
     {
-        public event EventHandler Notify;
+        public event EventHandler NotifyStarted;
+
+        public event EventHandler NotifyTrigger;
+
+        public void NotifyStartedSubscribers()
+        {
+            NotifyStarted?.Invoke(this, EventArgs.Empty);
```

Comment:
I don't really have experience with `EventHandler`, so I may be overlooking something, but what exactly are we invoking here? I see we add `NotifyHandler` for `NotifyTrigger`, but nothing for `NotifyStarted`.

Reply:
The `NotifyStarted` event is subscribed and unsubscribed in the new `ManualTriggerAsync` method in the tests. The `NotifyStartedSubscribers` method is invoked by the `ManualTrigger.StartAsync` method.

Reply:
Oops, I just realized that when I searched for it the `CollectionRulePipelineTests` diff hadn't loaded automatically, so it didn't show up - I see it now. I'll close this thread